doc/userguide/install.rst

```diff
@@ -66,52 +66,102 @@ Dependencies
 
 For Suricata's compilation you'll need the following libraries and their development headers installed::
 
-  libjansson, libpcap, libpcre2, libmagic, zlib, libyaml
+  libjansson, libpcap, libpcre2, libyaml, zlib
 
 The following tools are required::
 
-  make gcc (or clang) pkg-config
-
-For full features, also add::
-
-  libgeoip, liblua5.1, libhiredis, libevent
+  make gcc (or clang) pkg-config rustc cargo
 
 Rust support::
 
   rustc, cargo
 
-  Not every distro provides Rust packages yet. Rust can also be installed
-  directly from the Rust project itself::
+  Some distros provide outdated/don't provide Rust packages.
+  Rust can also be installed directly from the Rust project itself::
 
-  https://www.rust-lang.org/en-US/install.html
+    1) Install Rust https://www.rust-lang.org/en-US/install.html
+    2) Install cbinggen - cargo install --force cbindgen
+    3) Make sure the cargo path is within your PATH environment
+        e.g. echo 'export PATH=”${PATH}:~/.cargo/bin”' >> ~/.bashrc
+        e.g. export PATH="${PATH}:/root/.cargo/bin"
 
 Ubuntu/Debian
 """""""""""""
 
 Minimal::
 
-    apt-get install build-essential libpcap-dev   \
-                    libyaml-0-2 libyaml-dev pkg-config zlib1g zlib1g-dev \
-                    make libmagic-dev libjansson libjansson-dev libpcre2-dev
+    # Installed Rust and cargo as indicated above
+    apt-get install build-essential git libjansson-dev libpcap-dev \
+                    libpcre2-dev libtool libyaml-dev make pkg-config zlib1g-dev
+    cargo install --force cbindgen
```

Comment:
modern versions of distros package a sufficiently new cbindgen

Reply:
changed to 
2) Install cbindgen - if the cbindgen version is not found in the repository
       or is outdated, it can be installed as: cargo install --force cbindgen